### ISSUE TITLE
#2

### ISSUE THREAD

**Issue by kjell** [kjell/kombucha#2]
Time to start fresh.
## New scoby from Amelia.

I picked it up last night and spilled it all over my backpack. Sadly I lost a lot of the kombucha that should've gone towards spiking this batch. I biked it home (fingers crossed that it didn't freeze) and started it last night.
## Tea from Verdant

Zhu Rong Yunnan Black
1 filter bag packed full : 23g
Bowl : 10 cups of 180° water (tea says it should be 205, next time I'll heat it another way)

2 ounces. I used steeping bag packed full. Steeped two glass bowls for about 5 minutes each, in sequence. The woman I talked to at Verdant said I should use 4oz of tea in one batch, and that blew my mind. If that's true I've always made it far too weak, this time too. Their tea is fancy and expensive so I didn't want to blow it all on my first batch.

I stirred 4 cups of sugar into 20 cups of tea. Once dissolved I filled the jar to just under the shoulder with cold, carbon filtered water, stirred once or twice, and dumped in the scoby.
## Vessel

Renee gave me a sweet 3 gallon glass jar with a spigot, so it went in that. The spigot has both metal and plastic, neither of which are supposed to be great for kombucha. I'm hoping the ease of pouring out a taste daily will tighten the feedback loop.
## Brew

I'm going to keep it where I walk past and look at it a few times a day. For now it's by the door. I plan to taste a bit every day and watch for the top to film over.

**Comment by kjell** [comment]
It's starting to do stuff. When I first put the scoby as three small disks, they mostly floated. The next day they'd sunk to the bottom. I checked this AM and they're now floating again, and a definite skin is starting to form across the top of my jar. So it looks like it's working…

![2014-01-28 08 53 25](https://f.cloud.github.com/assets/1378/2026317/eb8d6e08-888e-11e3-9edd-b2352d07b47e.jpg)
![2014-01-28 08 53 04](https://f.cloud.github.com/assets/1378/2026318/eb8e753c-888e-11e3-8551-9897ee5b85b4.jpg)

Kombucha is weird.

**Comment by kjell** [comment]
I've been taking tablespoon sized sips a few times a day. Up until yesterday it tasted like sugary mild tea, now it definitely doesn't. It's turned a acidic and a tad effervescent. I'm excited to bottle some and see if the pressure builds.
- [x] bottle some

The scoby so far is a clear-ish film covering the surface, with the snot-looking bits growing independently and trapping air bubbles. The surface that's covered in scoby wiggles a bit like jello when I flick the side of the jar.

**Comment by kjell** [comment]
I filled two 16oz bottles. The bottle with a blank top has 1tsp of sugar in it.

![2014-01-31 19 08 56](https://f.cloud.github.com/assets/1378/2056482/9adc5d74-8add-11e3-8499-b78fcddaf6e1.jpg)

The scoby is still mostly clear. Pulling two bottles dropped the 'water level' an inch or two, and the scoby stuck to one side of the jar. The side towards the light, if that makes any difference. I forgot about keeping it in a dark place…
- [x] check on kombucha and light exposure

**Comment by kjell** [comment]
> I bottled 8 x 16oz two nights ago. I did one test the night before with lemon juice and ginger, and it turned out pretty well. I gave some to Eric W to test and he said it was good. I drank the rest of the bottle.

I cracked the first bottle from "this batch" today. It fizzed like crazy. Quite a bit poured over into the sink. It was flavored ginger lemon and tasted great. I had a few people over for brunch and they all also seemed to like it!

I went for the bottle that looked the most carbonated based on the bulge in the plastic cap. Next I'm going to try a bottle of plain, but I currently can't get the cap off.
- [ ] refrigerate the bottles to keep them from exploding.
